### PR TITLE
Update bug_report.md

### PR DESCRIPTION
**Pull request by nammn**
### All Submissions:

* [ ] Have you opened an Issue before filing this PR?
* [ ] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

**Review by Julien-Ben — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -37,4 +37,11 @@ Add any other context about the problem here.
...
  - yaml definitions for your objects
- - log files for the operator and database pods
+ - The operator logs
+ - Your Custom Resource below assuming the pods are named `mongo-0`
+ - The Pod logs:
+   - `kubectl logs mongo-0`
+ - The agent health status of the faulty members:
+   - `kubectl exec -it mongo-0 -c mongodb-agent -- cat /var/log/mongodb-mms-automation/healthstatus/agent-health-status.json`
+ - The agent logs of the faulty members:
+   - `kubectl exec -it mongo-0 -c mongodb-agent -- cat /var/log/mongodb-mms-automation/automation-agent-verbose.log`
```
Comment (lsierant) [inline .github/ISSUE_TEMPLATE/bug_report.md:47]: do we set debug log level by default? If not then let's add the log file without -verbose here as well

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -37,4 +37,11 @@ Add any other context about the problem here.
 If possible, please include:
  - `kubectl describe` output
  - yaml definitions for your objects
- - log files for the operator and database pods
+ - The operator logs
+ - Your Custom Resource below assuming the pods are named `mongo-0`
```
Comment (lsierant) [inline .github/ISSUE_TEMPLATE/bug_report.md:41]: this sentence is not clear to what exactly you refer here

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (nammn) [reply .github/ISSUE_TEMPLATE/bug_report.md:47]: I believe we do, but lets add both to be safe!

Comment (nammn) [reply .github/ISSUE_TEMPLATE/bug_report.md:58]: I added the change, let me know what you think!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (nammn) [reply .github/ISSUE_TEMPLATE/bug_report.md]: see my change https://github.com/mongodb/mongodb-kubernetes-operator/pull/1483/commits/61b65eb993bcf9672e5a49f7abb60ec0a11dd8d0

Comment (nammn) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I see, added the changes

**Pull request merged**